whois-api/src/main/java/net/ripe/db/whois/api/httpserver/hazelcast/WhoisHazelcastManagerFilter.java

```diff
@@ -0,0 +1,132 @@
+package net.ripe.db.whois.api.httpserver.hazelcast;
+
+import com.google.common.base.Joiner;
+import com.hazelcast.collection.ISet;
+import com.hazelcast.core.HazelcastInstance;
+import jakarta.servlet.Filter;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.FilterConfig;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.ServletRequest;
+import jakarta.servlet.ServletResponse;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import net.ripe.db.whois.api.httpserver.JettyBootstrap;
+import net.ripe.db.whois.common.ip.IpInterval;
+import net.ripe.db.whois.common.ip.Ipv4Resource;
+import net.ripe.db.whois.common.ip.Ipv6Resource;
+import org.eclipse.jetty.http.HttpStatus;
+import org.eclipse.jetty.util.StringUtil;
+import org.eclipse.jetty.util.annotation.ManagedOperation;
+import org.slf4j.Logger;
+import org.springframework.jmx.export.annotation.ManagedResource;
+
+import java.io.IOException;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+@ManagedResource(objectName = JettyBootstrap.BLOCK_LIST_JMX_NAME)
+public class WhoisHazelcastManagerFilter implements WhoisHazelcastManagerFilterMBean, Filter {
```

Comment:
Same name that the Interface without MBean to avoid the issue
Class net.ripe.db.whois.api.httpserver.x is not a JMX compliant MXBean

Reply:
no applicable after last refactor